_dev/src/store/modules/product-feed/actions.ts

```diff
@@ -21,6 +21,7 @@ import MutationsAppTypes from '../app/mutations-types';
 import ActionsTypes from './actions-types';
 import HttpClientError from '../../../utils/HttpClientError';
 import countriesSelectionOptions from '../../../assets/json/countries.json';
+import getDataFromLocalStorage from '../../../utils/LocalStorage';
```

Comment:
We could think about other methods later, can we have a name import?

```suggestion
import { getDataFromLocalStorage } from '../../../utils/LocalStorage';
```